src/containers/location/providers/LocationSearchBar.js

```diff
@@ -35,6 +35,15 @@ import { STATE, PROVIDERS } from '../../../utils/constants/StateConstants';
 
 const { media } = StyleUtils;
 
+/* placeholder color needs to be darker to provide more contrast between text and background */
+const getPlaceholderTheme = (theme) => ({
```

Comment:
I think this should be just `getTheme`. Also, some form of this change should also make its way into LUK.